### DISCUSSION TITLE
Welcome to EffiTrack Discussions!

### DISCUSSION THREAD

**Discussion by Cibiyanna26** [Cibiyanna26/EffiTrack#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aswathcm29** [comment]
That's great to hear, @Cibiyanna26 ! 🚀

Let's make this community thrive together! 💪

**Comment by muruga21** [comment]
yooo

**Comment by muruga21** [comment]
![image](https://github.com/Cibiyanna26/EffiTrack/assets/112178257/117ceb7f-1d9f-4fa8-897e-dc822c5120c2)
we need update the version regularly
